src/main/java/de/teamlapen/vampirism/entity/action/EntityActionManager.java

```diff
@@ -0,0 +1,28 @@
+package de.teamlapen.vampirism.entity.action;
+
+import com.google.common.collect.Lists;
+import de.teamlapen.vampirism.api.entity.actions.IEntityAction;
+import de.teamlapen.vampirism.api.entity.actions.IEntityActionManager;
+import de.teamlapen.vampirism.core.VampirismRegistries;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.registries.IForgeRegistry;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class EntityActionManager implements IEntityActionManager {
+    public static int ID = 0;
+    public static final Map<Integer, IEntityAction> actionsByID = new HashMap<>();
```

Comment:
Why do you use an integer map. All actions should have a registered registry name available and should also be element of VampirismRegistries.ENTITYACTIONS)